src/predict.jl

```diff
@@ -26,18 +26,26 @@ Predict response for new data.
     as such may use a large amount of memory when `newdata` is large.
 
 The keyword argument `new_re_levels` specifies how previously unobserved
-values of the grouping variable are handled. Possible values are
-`:population` (return population values, i.e. fixed-effects only),
-`:missing` (return `missing`), `:error` (error on this condition;
-the error type is an implementation detail). If you want simulated values
-for unobserved levels of the grouping variable, consider the
-[`simulate!`](@ref) and `simulate` methods.
+values of the grouping variable are handled. Possible values are:
+
+- `:population`: return population values for the relevant grouping variable.
+   In other words, treat the associated random effect as 0.
+   If all grouping variables have new levels, then this is equivalent to the
```

Comment:
```suggestion
   If all grouping variables have new levels, then this is equivalent to
```
Otherwise it's "the just the"